app/src/main/kotlin/co/netguru/baby/monitor/client/feature/analytics/AnalyticsManager.kt

```diff
@@ -0,0 +1,69 @@
+package co.netguru.baby.monitor.client.feature.analytics
+
+import android.app.Activity
+import androidx.core.os.bundleOf
+import com.google.firebase.analytics.FirebaseAnalytics
+import timber.log.Timber
+
+class AnalyticsManager(
+    private val firebaseAnalytics: FirebaseAnalytics
+) {
+    fun setCurrentScreen(activity: Activity, screenName: String) {
+        firebaseAnalytics.setCurrentScreen(activity, screenName, null)
+        Timber.d("$SCREEN $screenName")
+    }
+
+    fun logEvent(eventName: String) {
+        firebaseAnalytics.logEvent(eventName, null)
+        Timber.d("$EVENT $eventName")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/3bf76ac46cfa16ed8786cf59815392e8654387bd/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/analytics/AnalyticsManager.kt#L18"></span>Multiple occurrences of the same string literal within a single file detected.</td>
    </tr>
  </tbody>
</table>